### PR TITLE
New slotmachine to waste your money

### PR DESCRIPTION
**Pull request by seekin4u**
## About The Pull Request

New thing that consume your credits

## Why It's Good For The Game

More fun

## Changelog
:cl:
add: Added new thing - slotmachine.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/circuitboards/machinery/bar.dm [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/obj/item/weapon/circuitboard/slot_machine
+	name = T_BOARD("Slotmachine")
+	build_path = /obj/machinery/slotmachine
+	board_type = "machine"
+	origin_tech = list(TECH_DATA = 2)
+	req_components = list(
+		/obj/item/weapon/stock_parts/matter_bin = 1,
+		/obj/item/weapon/stock_parts/manipulator = 1,
+		/obj/item/weapon/stock_parts/console_screen = 1
+	)
```
Comment (SpadesSlick) [inline code/game/objects/items/weapons/circuitboards/machinery/bar.dm:10]: Please add new line.

**Comment by SpadesSlick** [comment]
Was it tested? Where is a map change?

**Comment by SpadesSlick** [comment]
Please include it on map.

**Review by drexample — CHANGES_REQUESTED** [review]
Add it on map and satisfy the required changes.

**Comment by drexample** [comment]
Closing in 3 days if no update.
@seekin4u

**Comment by fernandos13** [comment]
i'll map it hold a sec. and make a PR for his code

**Comment by seekin4u** [comment]
Can't do map PR atm, have no chance to do it on my work machine, you are free to do wtf you want with this pr

**Pull request merged**